elgg/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
+
+services:
+  db:
+    image: 'mariadb'
+    volumes:
+      - mariadb-data:/var/lib/mysql
+    environment:
+      - MYSQL_USER=$DATABASE_USER
+      - MYSQL_PASSWORD=$DATABASE_PASSWORD
+      - MYSQL_DATABASE=$DATABASE_NAME
+      - MARIADB_RANDOM_ROOT_PASSWORD=1
+    networks:
+      - elgg-network
+
+  elgg:
+    image: 'calteknet/elgg:latest'
+    user: ctn
+    ports:
+      - 9000
+    depends_on:
+      - db
+    networks:
+      - elgg-network
+    volumes:
+      - static-data:/opt/static
+      - elgg-data:/opt/data
+  
+  nginx:
+    image: 'localhost/calteknet/nginx:latest'
```

Comment:
Update image reference @thatnerdjosh

Reply:
Sorry I am not familiar with using this review of code. I figure that's why you have not done anything else to it. I imagine it would be safe to update the image with this code. I will see if I can get someone else that knows more about doing this to take a look. But meanwhile if you can continue the work you have done on this I think elgg is a good resource for projects Hakeem is telling me about, even today.